### DISCUSSION TITLE
咨询Kyuubi 基于Zk高可用方案，使用JDBC 连接Doris后，session宕机恢复的问题

### DISCUSSION THREAD

**Discussion by zhangbinzaifendou** [apache/kyuubi#6858] [General]
Kyuubi 基于Zk高可用方案，使用JDBC 连接Doris后，看代码中每个kyuubi维护自己的session信息且保存在内存中，如果Doris任务执行时间较长，希望通过session中的OperationHandle kill doris sql 作业，宕机后如何保证session相关的信息能够正常恢复。

**Comment by pan3793** [comment]
Currently, the interactive session state is stored in each Kyuubi server's memory, and it can not be restored after restarting.
